src/Bracket_60_1_A/sql/world/progression_60_1_A_spell_dbc.sql

```diff
@@ -0,0 +1,4 @@
+(20476,0,0,0,0,0,0,0,0,64,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,101,0,0,60,60,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,-1,0,0,2,98,0,1,35,0,0,0,0,2099,332,0,0,0,0,22,1,0,30,0,0,19,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,5679,0,37,0,0,'Explosion','','','','','','','','',0,0,0,0,0,'Pizza Hawaii',0,16712190,'','','','','','','','','',0,0,0,0,0,0,0,16712190,'Causes the target to explode,dealing $s1 Fire damage to all nearby allies.','','','','','','','','',0,0,0,0,0,0,0,16712190,'','','','','','','','','',0,0,0,0,0,0,0,16712190,0,0,0,0,0,0,0,0,0,1,1,0,1,1,1,0,0,0,0,0,0,4,0,0,0,1,1,1,0,0);
```

Comment:
```suggestion
(20476,0,0,0,0,0,0,0,0,64,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,101,0,0,60,60,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,-1,0,0,2,98,0,1,35,0,0,0,0,2099,332,0,0,0,0,22,1,0,30,0,0,19,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,5679,0,37,0,0,'Explosion','','','','','','','','',0,0,0,0,0,'Pizza Hawaii',0,16712190,'','','','','','','','','',0,0,0,0,0,0,0,16712190,'Causes the target to explode,dealing $s1 Fire damage to all nearby allies.','','','','','','','','',0,0,0,0,0,0,0,16712190,'','','','','','','','','',0,0,0,0,0,0,0,16712190,0,0,0,0,0,0,0,0,0,1,1,0,1,1,1,0,0,0,0,0,0,4,0,0,0,1,1,1,0,0);

```
add newline at the end of file